library_registry/admin/routes.py

```diff
@@ -10,59 +13,85 @@
     'admin', __name__,
     template_folder='templates')
 
+
 @admin.route('/admin/', strict_slashes=False)
 def admin_view():
     return current_app.library_registry.view_controller()
 
+
 @admin.route('/admin/log_in', methods=["POST"])
 @returns_problem_detail
 def log_in():
     return current_app.library_registry.admin_controller.log_in()
 
+
+@admin.route('/admin/log_in_w_token', methods=['POST'])
+@returns_problem_detail
+def log_in_w_token():
+    return current_app.library_registry.admin_controller.log_in_w_token()
```

Comment:
Would it be a good idea to unify this with the original `/log_in` method? Hypothetically we could set a session variable _and_ create tokens and therefore allow interaction using either mechanism?

Or maybe this could accept a parameter that specifies the type of authentication we're doing? That way we could let applications choose but not require the additional endpoint.

Can see arguments for all sides!

Reply:
Great idea, I have extended the log_in function to check for a 'jwt': boolean in the form. And handle both routes.

Reply:
Hey Bob. Your approach is a good first attempt at this. The one downside is that it assumes the client is going to have a toggle or box to check if they want jwt, and that isn't really something we can expect. Have you tried the parameter approach? The client can pass in how they want to authenticate to the `/admin/log_in` route, and you'll just need to pass that to your `log_in()` method in the controller and handle there (it looks like you already have the shape of it all in place there...great!).

Reply:
Great idea. I have updated accordingly.